.github/workflows/vale-lint.yml

```
@@ -0,0 +1,40 @@
+name: Validate content with Vale
+
+on:
+  pull_request:
+    paths:
+      - 'platform/**/*.mdx'  # Include .mdx files in platform folder and subfolders
+      - 'platform/**/*.md'   # Include .md files in platform folder and subfolders
```

I think these should also contain:


```suggestion
      - 'vcluster/**/*.mdx'  # Include .mdx files in platform folder and subfolders
      - 'vcluster/**/*.md'   # Include .md files in platform folder and subfolders
```

---

I was thinking to start from the `platform` folder only and see how the setup works before extending to other folders. I'm expecting a few _fast-follow_ PRs before we get everything as we want.